ajax/libs/chocolat/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "chocolat",
+  "filename": "js/jquery.chocolat.min.js",
+  "version": "0.4.13",
+  "description": " Responsive jQuery Lightbox Plugin",
+  "dependencies": {
+    "jquery": "~1.8.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nicolas-t/Chocolat.git"
+  },
+  "keywords": [
+    "jquery",
+    "lightbox",
+    "plugin",
+    "responsive"
+  ],
+  "author": "Nicolas Turlais",
+  "homepage": "http://chocolat.insipi.de",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nicolas-t/Chocolat.git",
+    "basePath": "dist",
+    "files": [
+      "!(demo*|index*)/*"
```

Comment:
@PeterDaveHello I'm not sure it's ok or not, but this can grab the right files. Or the ```!(demo)**!(html)``` can also work. Please help us.